core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -524,7 +524,7 @@ public EnumSet<Block.VerifyFlag> getBlockVerificationFlags(final Block block,
     public EnumSet<Script.VerifyFlag> getTransactionVerificationFlags(final Block block,
             final Transaction transaction, final VersionTally tally, final Integer height) {
         final EnumSet<Script.VerifyFlag> verifyFlags = EnumSet.noneOf(Script.VerifyFlag.class);
-        if (block.getTimeSeconds() >= NetworkParameters.BIP16_ENFORCE_TIME)
+        if (block.time().getEpochSecond() >= NetworkParameters.BIP16_ENFORCE_TIME)
```

Comment:
Shouldn't `BIP16_ENFORCE_TIME` be made available as a time type?

Reply:
We could, and perhaps should. But it's out of scope for this PR.